usage/test-framework/src/main/java/org/apache/brooklyn/test/framework/BaseTest.java

```diff
@@ -47,16 +47,16 @@
     ConfigKey<String> TARGET_ID = ConfigKeys.newStringConfigKey("targetId", "Id of the entity under test");
 
     /**
-     * The assertions to be made
+     * The assertions to be made.
      */
-    ConfigKey<List<Map<String, Object>>> ASSERTIONS = ConfigKeys.newConfigKey(
-            new TypeToken<List<Map<String, Object>>>() {},
-            "assert", "Assertions to be evaluated", new ArrayList<Map<String, Object>>());
+    ConfigKey<Object> ASSERTIONS = ConfigKeys.newConfigKey(Object.class, "assert", "Assertions to be evaluated",
+        new ArrayList<Map<String, Object>>());
```

Comment:
Personal preference for the default value of the config key to be immutable. Otherwise if someone modifies it, things get very strange (it changes the default for all other entities that use it, until Brooklyn is restarted!)

I'd therefore use `ImmutableList.of()` (presumably the generics don't matter, now that it's change to be a config key of type `Object`).